quantum/encoder.c

```diff
@@ -17,6 +17,12 @@
 
 #include "encoder.h"
 
+#ifdef SPLIT_KEYBOARD
+  // for memcpy
+  #include <string.h>
+#endif
+
+
```

Comment:
I don't think this ifdef is necessary.
The following one line is enough.
```c
#include <string.h>
```

Reply:
ya, removing the ifdef won't cause a compile error or anything, but seemed silly to include a define for all when it was only needed when using split encoders. I'll remove it if you feel it's worth it.

Reply:
I think that files of C standard can be imported unconditionally. But I respect your decision.

Reply:
Well, a good question is: is gcc compiling out the code when it's not used?   if so, then there is no harm in leaving it in, all the time.  Otherwise, yeah...

Reply:
Not seeing anything in the lst/o files that would indicate it is a problem leaving it in. Will remove it